manuscript/conclusion.md

```diff
@@ -81,11 +83,67 @@ As we saw in this book Open Source doesn't mean just release the code but let pe
+
+Two things are very interesting for the purpose of this book about what they are doing.
+
+### PublicCode.yml standard
+
+`
+A metadata description standard for public software that is easy to use both for developers and people with less technical background in order to make the software developed by Public Administrations and Public Organizations easily discoverable. 
+`
+
+In few words, a file that you can put in your git repository that is like an identity card that allow to index it with references to the maintainer and dependencies.  
+The idea is to improve the discoverability and if that project is suitable for a public administration and to use the standard also in other countries.
+
+## Catalogo del software di Riuso
+
+That in English "Reuse Software catalog" that Italian public administration can use it. As every big organization departments don't talk each others, also cities or region, but they have often the same needs, so why develop multiple times the same software?  
+
+This is the idea at the end, a way for public administration where often managers have no so much knowledge on IT to find software already available and open source developed by themselves or by others like LibreOffice for example.  
+
+This project uses the standard we saw to fetch the software list, but the website is more friendly as includes more information, not just a description but people that you can reach to ask question. It is something that for a bureaucratic organization is important and that in OSS is often ignored.  
+After all, if you are maintaining a project you don't want that people reach you for questions about if the software can do this or that, or if you can implement something just for them. Especially if it is someone that maybe will pay you...
+
+This is an example about how improve the awareness to organization that is not like the usual software alternative list, because can give more information that are valuable to the people in charge of decisions. 
+
+In a first world country there are limitations or issues like elsewhere but it is important to see that internet, public awareness and young people are changing the situation. Also, if the [Italian case](https://profilo.forumpa.it/doc/?file=2021/2021_FPA_Data_Insight_Lavoro_pubblico_2021.pdf&confirm=yes) seems that in the public government the people under 30 years old is the 4%. Especially in [School institution the age average is the 58% is over 50](https://www.truenumbers.it/eta-degli-insegnanti/) but in Ocse countries is the 35%.
+
+I know people in Italian universities that still asks questions about Floppy disks to students of psychology for example or that teach using computer with very old operative systems because the professors doesn't know how to use the modern ones.
```

Comment:
```suggestion
I know people in Italian universities that still asks questions about Floppy disks to students of psychology for example or that teach using computer with very old operative systems because the professors don't know how to use the modern ones.
```